components/CollectiveNavbar.js

```diff
@@ -386,78 +387,80 @@ const CollectiveNavbar = ({
       </InfosContainer>
 
       {/** Navbar items and buttons */}
-      <Container
-        position={['absolute', 'relative']}
-        display="flex"
-        justifyContent="space-between"
-        px={[0, Dimensions.PADDING_X[1]]}
-        width="100%"
-        background="white"
-      >
-        {isLoading ? (
-          <LoadingPlaceholder height={43} minWidth={150} mb={2} />
-        ) : (
-          <Container
-            flex="2 1 600px"
-            display={isExpended ? 'flex' : ['none', null, 'flex']}
-            css={{ overflowX: 'auto' }}
-            data-cy="CollectivePage.NavBar"
-            flexDirection={['column', null, 'row']}
-            height="100%"
-            borderBottom={['1px solid #e6e8eb', 'none']}
-          >
-            {sections.map(section => (
-              <MenuLinkContainer
-                key={section}
-                isSelected={section === selected}
-                onClick={() => {
-                  if (isExpended) {
-                    setExpended(false);
-                  }
-                  if (onSectionClick) {
-                    onSectionClick(section);
-                  }
-                }}
-              >
-                <MenuLink
-                  as={LinkComponent}
-                  collectivePath={collective.path || `/${collective.slug}`}
-                  section={section}
-                  label={i18nSection[section] ? intl.formatMessage(i18nSection[section]) : section}
-                />
-              </MenuLinkContainer>
-            ))}
-            {callsToAction.hasSubmitExpense && (
-              <MenuLinkContainer mobileOnly>
-                <MenuLink as={Link} route="createExpense" params={{ collectiveSlug: collective.slug }}>
-                  <FormattedMessage id="menu.submitExpense" defaultMessage="Submit Expense" />
-                </MenuLink>
-              </MenuLinkContainer>
-            )}
-            {callsToAction.hasContact && (
-              <MenuLinkContainer mobileOnly>
-                <MenuLink href={`mailto:hello@${collective.slug}.opencollective.com`}>
-                  <FormattedMessage id="Contact" defaultMessage="Contact" />
-                </MenuLink>
-              </MenuLinkContainer>
-            )}
-            {callsToAction.hasDashboard && collective.plan.hostDashboard && (
-              <MenuLinkContainer mobileOnly>
-                <MenuLink as={Link} route="host.dashboard" params={{ hostCollectiveSlug: collective.slug }}>
-                  <FormattedMessage id="host.dashboard" defaultMessage="Dashboard" />
-                </MenuLink>
-              </MenuLinkContainer>
-            )}
-          </Container>
-        )}
-        <div>
-          <CollectiveCallsToAction
-            display={['none', null, 'flex']}
-            collective={collective}
-            callsToAction={callsToAction}
-          />
-        </div>
-      </Container>
+      {!onlyInfos && (
```

Comment:
It's only indenting change when introducing this `onlyInfos` prop. I needed it for the Host Dashboard.